homeassistant/components/sensor/rmvtransport.py

```diff
@@ -90,6 +91,13 @@
                 next_departure.get(CONF_MAX_JOURNEYS),
                 next_departure.get(CONF_NAME),
                 timeout))
+
+    for sensor in sensors:
+        await sensor.async_update()
```

Comment:
We could use `asyncio.wait`.
```py
tasks = [sensor.async_update() for sensor in sensors]
if tasks:
    await asyncio.wait(tasks)
if not all(sensor.data.departures for sensor in sensors):
    raise PlatformNotReady
```

Reply:
We gain efficiency by using asyncio.wait but on the other hand lose efficiency by finishing all I/O before checking the result. :man_shrugging:

Reply:
That is neat.

Reply:
You win some, you loose some, huh?

Reply:
I think either approach is fine.